### PR TITLE
Hardsuit helmet toggling now has a delay

### PR DESCRIPTION
**Pull request by ChangelingRain**
Oh, god damn it.

### CODE REVIEW

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -85,6 +85,12 @@
 		jetpack = new jetpack(src)
 	..()
 
+/obj/item/clothing/suit/space/hardsuit/attack_self(mob/user)
+	if(world.time <= user.next_move)
```
Comment (optimumtact) [inline code/modules/clothing/spacesuits/hardsuit.dm]: this is checked before this code is ever run, you don't need it

**Review by optimumtact — CHANGES_REQUESTED** [review]

**Comment by Supermichael777** [comment]
what bug does his fix?

**Review by optimumtact — APPROVED** [review]

**Comment by PKPenguin321** [comment]
this was a long time coming

arm mods need this too since they also make the noise

**Comment by Shapsy** [comment]
SHINGSHINGSHINGSHING

**Pull request merged**